uefi/Cargo.toml

```diff
@@ -43,7 +43,8 @@ uguid.workspace = true
 cfg-if = "1.0.0"
 ucs2 = "0.3.3"
 uefi-macros = "0.19.0"
-uefi-raw = "0.12.0"
+# TODO: switch back to public version before merging.
+uefi-raw.path = "../uefi-raw"
```

Comment:
this change is not needed and can be removed. in the top-level Cargo.toml we use the `[patch.crates-io]` mechanism which ensures that whatever version one references from crates.io, it is replaced by the local workspace member